src/pages/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import AnnouncementList from "../components/AnnouncementList";
+import SdkGroupsList from "../components/SdkGroupsList";
+
+export default function Homepage() {
+  return (
+    <Layout
+      title="All Guides"
+      description="Guides and references to help you integrate the Helpshift SDK into
+      your app"
```

Comment:
Minor - This shouldn't be a new line. You can ignore these comments and fix them later. I am okay with that.

Reply:
Done: 758f52d